Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
@daschult - the scenario it should support is:

1. Install Dependency Telemetry Module.
2. Call `(new DependencyTrackingTelemetryModule()).Initializer(TelemetryConfiguraiton.Active); ` implicitly or explicitly.
3. See dependencies in the portal.

@dnduffy - for now - browse thru the collection of telemetry modules in services. Find `DependencyTrackingTelemetryModule`. Set the additional configuration settings like the list of excluded modules.